core/types/block.go

```diff
@@ -330,6 +330,9 @@ func (b *Block) Extra() []byte            { return common.CopyBytes(b.header.Ext
 
 func (b *Block) Header() *Header { return CopyHeader(b.header) }
 
+// Body returns the non-header content of the block.
+func (b *Block) Body() *Body { return &Body{b.transactions, b.uncles} }
```

Comment:
Not sure, but it sounds like a waste of memory to me.

The change to "immutable" (copying) accessors was meant to be
made up for by reusing the memory obtained through the accessors.
But that turned out to be weird-looking so we don't do it.